rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java

```diff
@@ -0,0 +1,102 @@
+package gov.cms.qpp.conversion.api.security;
+
+import com.google.common.collect.Sets;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.web.authentication.www.BasicAuthenticationFilter;
+
+/**
+ * Filter for checking the Json Web Token (JWT) for the correct Authorization
+ */
+public class JwtAuthorizationFilter extends BasicAuthenticationFilter {
+	private static final String HEADER_STRING = "Authorization";
+	private static final String TOKEN_PREFIX = "Bearer ";
+	private static final Set<String> VALID_ORG_IDS = Sets.newHashSet("fb1778dd-a5e3-42c8-836f-47654e003fab",
+			"ef33b1f6-842a-4519-9109-2ca2ed728ceb", "45624754-cce1-47ea-a7b1-02a9b9e19c2a");
```

Comment:
Should we externalize these into an env variable?

Reply:
I thought that as well, but @halprin argued that it would be fine to have them hardcoded because this is a simple check to verify their organization and wouldn't be helpful in generating a token to validate through the API gateway. I'm down for either.

Reply:
I think an environment variable would be most appropriate.

Reply:
👍  Majority rules. I'll change this into a new environment variable.